app/uk/gov/hmrc/exports/services/mapping/goodsshipment/consignment/GoodsLocationBuilder.scala

```diff
@@ -71,27 +61,9 @@ class GoodsLocationBuilder @Inject()(countriesService: CountriesService) extends
   private def createAddress(goods: GoodsLocation): Consignment.GoodsLocation.Address = {
     val goodsAddress = new Consignment.GoodsLocation.Address()
 
-    goods.addressLine.foreach { value =>
-      val line = new AddressLineTextType()
-      line.setValue(value)
-      goodsAddress.setLine(line)
-    }
-
-    goods.city.foreach { value =>
-      val city = new AddressCityNameTextType
-      city.setValue(value)
-      goodsAddress.setCityName(city)
-    }
-
-    goods.postCode.foreach { value =>
-      val postcode = new AddressPostcodeIDType()
-      postcode.setValue(value)
-      goodsAddress.setPostcodeID(postcode)
-    }
-
     if (goods.country.nonEmpty) {
       val countryCode = new AddressCountryCodeType
```

Comment:
So we were passing the country name and then looking up the code in the backend to pass to xml?

Reply:
Nope, we're passing country code from the frontend. This is the reason that I'm not manipulating data here.

Previously we had country as dropdown, actually it's one input with code e.g. `GBAUFXTFXTFXT` so we're passing two first characters to the backend and just put this into the payload.

Reply:
Yes.  I was just commented on what we used to do.  I couldn't find the facepalm icon!